integration_tests/src/main/python/data_gen.py

```diff
@@ -708,13 +745,16 @@ def gen_scalars_for_sql(data_gen, count, seed=0, force_no_nulls=False):
 # TODO add in some array generators to this once that is supported for sorting
 # a selection of generators that should be orderable (sortable and compareable)
 orderable_gens = [byte_gen, short_gen, int_gen, long_gen, float_gen, double_gen,
-        string_gen, boolean_gen, date_gen, timestamp_gen, null_gen]
+        string_gen, boolean_gen, date_gen, timestamp_gen, null_gen] + decimal_gens
```

Comment:
Do the sort tests work with this change?

Reply:
It doesn't affect sort tests as they don't use `orderable_gens` instead use `orderable_gen_classes`

Reply:
It doesn't affect sort tests as they don't use `orderable_gens` instead use `orderable_gen_classes`